### PR TITLE
Post-demo fixes

### PR DESCRIPTION
**Pull request by AnonymousRandomPerson**
-Nerfed BBFly and EmberFox by making them move slower.
-Fixed malloc bullet explosions getting progressively larger.
-Fixed clipping through destroyable blocks after a cutscene reset.
-Fixed grass block size in Kirby level.
-Fixed camera bounds in boss levels.
-Fixed destroyer bullet cutscene (adjusted for reduced destroyer range).
-Fixed bullets hitting the instructions trigger collider.
-Fixed player standing on Doodle beam.
-Fixed negative player health on display.

### CODE REVIEW

File: Assets/Scripts/Managers/PlayerDeathHandler.cs [hunk 1]
```diff
@@ -68,6 +68,7 @@ void Update()
                     {
                         destroyed.GetComponent<SpriteRenderer>().enabled = true;
                         destroyed.GetComponent<Collider2D>().enabled = true;
+                        destroyed.layer = LayerMask.NameToLayer("Default");
```
Comment (JonathanHunter) [inline Assets/Scripts/Managers/PlayerDeathHandler.cs:71]: Why is this line here? You shouldn't need to change the layer of anything during reset.

Comment (AnonymousRandomPerson) [inline Assets/Scripts/Managers/PlayerDeathHandler.cs:71]: The destroyer bullet changes the object's layer to "Destroyed," so that line is needed to reset this change. Not doing this is what caused the player to go through the blocks.

Comment (JonathanHunter) [inline Assets/Scripts/Managers/PlayerDeathHandler.cs:71]: Ok thats fine.  merging
